cmd/kube-scheduler/app/options/options.go

```diff
@@ -124,7 +124,7 @@ func splitHostIntPort(s string) (string, int, error) {
 	if err != nil {
 		return "", 0, err
 	}
-	portInt, err := strconv.Atoi(port)
+	portInt, err := utilnet.ParsePort(port, true)
```

Comment:
Out of curiosity why is the allowZero set to true here?

Reply:
The function returns 0 as a port number, so I think that zero is a valid port number.

Reply:
0 was previously allowed, when used as a bind port, that means a free port is auto-selected

Reply:
This might make a good comment